assets/css/custom/program.scss

```diff
@@ -221,6 +221,28 @@ table td:nth-child(1) {
   margin-top: 10px;
 }
 
+.program-name {
+  min-width: 30px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
No, this was just in the style property in the program name div and I wanted to use the same styling for the translated program name so I thought it would be better to create a new class to encapsulate the styling.